### DISCUSSION TITLE
What happens when episodes are deleted from the RSS?

### DISCUSSION THREAD

**Discussion by dreamflasher** [jeff-hughes/shellcaster#53] [Q&A]
I am migrating and thus new to shellcaster, and one thing I'd appreciate clarification on is how deletions in the RSS feed are handled. Will this also be removed from the DB and disk or will those be untouched (I'd prefer the latter)?

**Comment by capturts** [comment]
(I'm not the developer)

The episodes stay in the database. If they are downloaded, you can listen at will. If they haven't been downloaded there is a chance that the podcast creator will delete the mp3s from their server and they will not be able to be retrieved.
